addons/discovery-app/@{{ cookiecutter.project_slug }}/deploy/app/helm_release.tf

```diff
@@ -0,0 +1,97 @@
+locals {
+  iam_role_arn       = data.terraform_remote_state.infra_local.outputs.iam_eks_role_arn
+  ecr_repository_url = data.terraform_remote_state.infra_local.outputs.ecr_repository_url
+  branch             = trim(replace(substr(lower(var.branch), 0, 53), "/", "-"), "-")
+  app_name           = "${var.app_name}-${local.branch}"
+  namespace          = local.app_name
```

Comment:
I would remove the namespace local and use local.app_name when needed (detail but here app_name and namespace have the same value)

Reply:
yes, this will be changed and aligned to discovery-gazette (as we discussed this morning).